lib/elixir/lib/map.ex

```diff
@@ -371,6 +371,26 @@ defmodule Map do
     :maps.update(key, value, map)
   end
 
+  @doc """
+  Replaces the value under `key` using the given function only if
+  `key` already exists in `map`.
```

Comment:
Can you add the same comment I suggested above here on when this function is useful?